pynest/examples/arbor_cosim_example/arbor_proxy.py

```diff
@@ -31,8 +31,7 @@ def print_spike_array_d(to_print, force=False):
 
     # Assume that we received a spike array, no error checking
     print(print_prefix + "SPIKES: [", end='')
-    for spike in to_print:
-        msg =
+    for spike in to_print:       
```

Comment:
pep8 formatting, trailing whitespaces removed.
```suggestion
    for spike in to_print:
```